tests/frontend/test_ratings.py

```diff
@@ -10,6 +10,84 @@
 from pages.desktop.frontend.reviews import Reviews
 
 
+@pytest.mark.serial
+@pytest.mark.nondestructive
+def test_rating_term(selenium, base_url, variables):
+    # this test checks that if the rating has no text, the term used in messages is 'rating'
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    addon.login('rating_user')
+    addon.ratings.rating_stars[3].click()
+    addon.ratings.wait_for_delete_link()
+    assert 'rating' in addon.ratings.delete_rating_link.text
+    addon.ratings.delete_rating_link.click()
+    assert 'rating' in addon.ratings.ratings_card_summary
+    assert 'rating' in addon.ratings.delete_confirm_button.text
+    assert 'rating' in addon.ratings.keep_review_button.text
+
+
+@pytest.mark.serial
+@pytest.mark.nondestructive
+def test_review_term(selenium, base_url, variables):
+    # this test checks that if the rating has text, the term used in messages is 'review'
+    extension = variables['detail_extension_slug']
+    selenium.get(f'{base_url}/addon/{extension}')
+    addon = Detail(selenium, base_url).wait_for_page_to_load()
+    addon.login('rating_user')
+    # write text to rating from previous test
+    addon.ratings.write_a_review.click()
+    addon.ratings.review_text_input('nice add-on')
+    addon.ratings.submit_review()
+    assert 'review' in addon.ratings.delete_rating_link.text
+    addon.ratings.delete_rating_link.click()
+    assert 'review' in addon.ratings.ratings_card_summary
+    assert 'review' in addon.ratings.delete_confirm_button.text
+    assert 'review' in addon.ratings.keep_review_button.text
+    addon.ratings.click_delete_confirm_button()
```

Comment:
Maybe we can integrate this in `test_rating_with_text`

Reply:
Ok